### PR TITLE
add amin to jax.numpy namespace functions

### PR DESCRIPTION
**Pull request by mostfa29**

### CODE REVIEW

**Comment by mostfa29** [comment]
hi @abdrahmandiab the formatting test error says 'there is no new line at the end of files" however if you can see the commit you can see there is a new line can you help me?

File: ivy/functional/frontends/jax/numpy/name_space_functions.py [hunk 1]
```diff
@@ -116,3 +116,7 @@ def var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False, *, where=Non
     if ivy.is_array(where):
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret.astype(dtype)
+
+
+def amin(a, axis=None, out=None, keepdims=None, initial=None, where=None):
+    return ivy.min(a, axis=axis, out=out, keepdims=keepdims)
```
Comment (abdrahmandiab) [inline ivy/functional/frontends/jax/numpy/name_space_functions.py:122]: Could you please add functionality to emulate the `where` and `initial` parameters as well? 

The `where` parameter can be handled using `ivy.where`, see examples in this file here on more details of how to implement this, or let me know if you need help! :)

Comment (abdrahmandiab) [reply ivy/functional/frontends/jax/numpy/name_space_functions.py:122]: See also [here](https://numpy.org/doc/stable/reference/generated/numpy.ufunc.reduce.html#numpy.ufunc.reduce) and [here](https://numpy.org/doc/stable/reference/generated/numpy.amin.html) for more info on the `initial` parameter.

Comment (abdrahmandiab) [reply ivy/functional/frontends/jax/numpy/name_space_functions.py:122]: the numpy frontends [`sum`](https://github.com/unifyai/ivy/blob/e02ae8eb46a20bcda72688fb3ac08080a33562f5/ivy/functional/frontends/numpy/mathematical_functions/sums_products_differences.py#L9) might also provide some guidance on how to handle the `initial` argument.

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py [hunk 2]
```diff
@@ -620,3 +620,43 @@ def test_jax_numpy_einsum(
...
+        max_axis=0,
+        min_num_dims=1,
+        force_int_axis=True,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.jax.numpy.amin"
+    ),
+    keep_dims=st.booleans(),
+)
+def test_jax_numpy_amin(
```
Comment (abdrahmandiab) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:640]: the `where` and `initial` parameters should be added to testing as well.
The test for numpy frontends [`sum`](https://github.com/unifyai/ivy/blob/e02ae8eb46a20bcda72688fb3ac08080a33562f5/ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py#L37) might also be a useful reference here.

**Review by abdrahmandiab — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py (hunk 2 above)
Comment (mostfa29) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:640]: thanks a lot Abdarhman I will work to fix this issues.

**Comment by mostfa29** [comment]
Hey, I'm sorry I am currently very busy and I can't fix the issues.

**Pull request closed without merging**